pkg/crd/crds/backups-backup.schema.generated.json

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
Try to parse it to yaml - use k8s.io/apimachinery/pkg/util/yaml

Reply:
Done